docs/history-mode.md

```diff
@@ -0,0 +1,88 @@
+
+Thus all content in a live state (published, published but needs 2i review, withdrawn, removed) will have an association with a government as these will all have a date indicating when they were published on GOV.UK.
+
+For draft states (draft, submitted for review, scheduled) content is [associated with a government](https://github.com/alphagov/content-publisher/blob/5246e7ffa8a33d3b607531ed66829e4dba5cc5ec/app/services/edit_edition_service.rb#L30) based on whether the content is [backdated](https://github.com/alphagov/content-publisher/blob/5246e7ffa8a33d3b607531ed66829e4dba5cc5ec/app/models/edition.rb#L139)) and whether it is a new edition of published content. For example:
+
+- A first edition of a document that is not backdated will not be associated with a government until it is published
+- A second edition of a document that is not backdated will be associated with the government from the publish date
+- A backdated edition will be associated with the government at the time of the backdating
+
+It's worth noting that backdating takes precedence over a publish date in determining which government.
+
+## How is content defined as political?
+Content can be defined as political. This is used as a shorthand way of saying that this content represents the policies of a particular government.
+
+By default the [political status of content is determined automatically](https://github.com/alphagov/content-publisher/blob/5246e7ffa8a33d3b607531ed66829e4dba5cc5ec/app/services/edit_edition_service.rb#L25) whenever the content is edited. The following [criteria](https://github.com/alphagov/content-publisher/blob/5246e7ffa8a33d3b607531ed66829e4dba5cc5ec/lib/political_edition_identifier.rb#L16) is used to determine political status:
+
+- What document type the content is
+- Whether it is associated with an organisation that publishes content that mostly represent the policies of the current government
+- Whether the content is tagged to a role appointment (we eventually hope to expand this to be whether the content is tagged to a minister of the respective government)
+
+There is an ability to override whether content is defined as political. This is restricted to specific users. These users can specify content is political or not and the previously mentioned criteria will no longer have an affect.
+
+## Who has the ability to change the attributes that affect history mode?
+
+An intention behind history mode is to restrict the ability of publishers to make edits to content once a government has changed. This is to preserve content in the state it was at the point of a government change.
+
+This is reflected by:
+
+- Only Managing editors with the `MANAGE_LIVE_HISTORY_MODE` permission have the ability to withdraw, unwithdraw, remove or create new editions of published content that is in history mode.
```

Comment:
You can remove the create new editions bit here as that's not true and is covered by the next bullet.